### DISCUSSION TITLE
Creating admin user from the command line

### DISCUSSION THREAD

**Discussion by papebarka** [solidusio/solidus#5085] [Troubleshooting] [answered]
Hi,

How is it possible to create a Solidus admin user from the command line?

Regards

**Comment by jarednorman** [comment, marked as answer]
Yep!

```ruby
Spree::User.create!(email: "admin@example.com", password: "yourpassword", spree_roles: Spree::Role.where(name: "admin"))
```

**Reply by benjaminwil** [reply]
In case the OP doesn’t have previous Rails experience: you’ll need to start a Rails console first: `bundle exec rails console`
